src/lib/__tests__/date.test.ts

```diff
@@ -1,51 +1,271 @@
 import { isNull } from "lodash"
 import moment, { Moment } from "moment"
-import { exhibitionPeriod, exhibitionStatus } from "lib/date"
+import {
+  dateRange,
+  exhibitionStatus,
+  datesWithHours,
+  datesWithDaysAndHours,
+  singleDateWithoutHour,
+  singleDate,
+  singleHour,
+  formattedOpeningHours,
+  datesAreSameDay,
+  formattedHours,
+} from "lib/date"
 
-describe("date", () => {
-  describe("exhibitionPeriod", () => {
-    it("includes the start and end date", () => {
-      const period = exhibitionPeriod(
-        moment("2011-01-01T00:00-0400"),
-        moment("2014-04-19T00:00-0400")
+describe("date formatting", () => {
+  describe("formattedHours", () => {
+    it("If within same am/pm", () => {
+      const period = formattedHours(
+        "2022-12-30T08:00:00+00:00",
+        "2022-12-30T10:00:00+00:00"
+      )
+      expect(period).toBe("8:00 – 10:00am")
+    })
+
+    it("If within across both am/pm", () => {
+      const period = formattedHours(
+        "2021-12-05T08:00:00+00:00",
+        "2022-12-30T17:00:00+00:00"
+      )
+      expect(period).toBe("8:00am – 5:00pm")
+    })
+  })
+
+  describe("datesAreSameDay", () => {
+    it("Dates are the same day", () => {
+      const period = datesAreSameDay(
+        "2022-12-30T20:00:00+00:00",
+        "2022-12-30T17:00:00+00:00"
+      )
+      expect(period).toBe(true)
+    })
+
+    it("Dates are not the same day", () => {
+      const period = datesAreSameDay(
+        "2021-12-05T20:00:00+00:00",
+        "2022-12-30T17:00:00+00:00"
+      )
+      expect(period).toBe(false)
+    })
```

Comment:
What is `datesAreSameDay` behaviour when times are each in different timezones? I think you can really only think of a ‘day’ when times are in the same TZ, as a time in one TZ could be a different day in another TZ in _real-time_.

Something that I could imagine is: 

```js
function datesAreSameDay(time1, time2, consideredInTZ) {
  // …
}
```